### PR TITLE
Support @x- attributes for operations

### PR DESCRIPTION
**Pull request by yutopp**
**Describe the PR**
Add support `@x-` attributes for operations in paths section. (https://swagger.io/docs/specification/2-0/swagger-extensions/)

**Relation issue**
https://github.com/swaggo/swag/issues/398. This PR introduces additional supports for it.

**Additional context**
(none)

Thanks.

### CODE REVIEW

File: testdata/simple2/docs/docs.go [hunk 1]
```diff
@@ -1,17 +1,19 @@
 // GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag at
-// 2019-06-25 15:43:55.76642234 +0200 CEST m=+2.220403193
+// 2019-07-15 21:17:57.384551624 +0900 JST m=+0.027691337
 
 package docs
 
 import (
 	"bytes"
+	"encoding/json"
```
Comment (yutopp) [inline testdata/simple2/docs/docs.go]: Diffs are appeared after running `make test` (Probably not related to changes by this PR...) Please let me know if there are some problems :pray:

Comment (easonlin404) [reply testdata/simple2/docs/docs.go]: It's ok for it.

**Comment by yutopp** [comment]
Thank you for your review. I fixed that. f85ce40

**Comment by easonlin404** [comment]
@yutopp Thanks four your contributions.

**Pull request merged**